packages/console/src/components/RadioGroup/index.module.scss

```diff
@@ -1,72 +1,96 @@
 @use '@/scss/underscore' as _;
 
 .radioGroup {
-  display: flex;
-  flex-wrap: wrap;
-  margin: 0 _.unit(-8) _.unit(-8) 0;
+  input[type='radio'] {
+    appearance: none;
+    position: absolute;
+    margin: 0;
+    width: 0;
+    height: 0;
+  }
 
-  > .radio {
-    position: relative;
-    flex: 1;
-    min-width: 180px;
-    max-width: 220px;
-    padding: _.unit(5);
+  &.card {
     display: flex;
-    flex-direction: column;
-    border-radius: _.unit(4);
-    border: 1px solid var(--color-neutral-90);
-    outline: none;
-    user-select: none;
-    cursor: pointer;
-    transition: border 0.2s ease-in-out;
-    margin: 0 _.unit(8) _.unit(8) 0;
-
-    &:focus,
-    &:hover {
-      border-color: var(--color-primary);
-    }
+    flex-wrap: wrap;
+    margin: 0 _.unit(-8) _.unit(-8) 0;
 
-    svg {
-      opacity: 0%;
-      position: absolute;
-      right: _.unit(5);
-      top: _.unit(5);
-    }
+    > .radio {
+      position: relative;
+      flex: 1;
+      min-width: 180px;
+      max-width: 220px;
+      padding: _.unit(5);
+      display: flex;
+      flex-direction: column;
+      border-radius: _.unit(4);
+      border: 1px solid var(--color-neutral-90);
+      outline: none;
+      user-select: none;
+      cursor: pointer;
+      transition: border 0.2s ease-in-out;
+      margin: 0 _.unit(8) _.unit(8) 0;
 
-    &.checked {
-      border-color: var(--color-primary);
-      outline: 1px solid var(--color-primary);
+      &:focus,
+      &:hover {
+        border-color: var(--color-primary);
+      }
 
       svg {
-        opacity: 100%;
+        opacity: 0%;
+        position: absolute;
+        right: _.unit(5);
+        top: _.unit(5);
+      }
+
+      &.checked {
+        border-color: var(--color-primary);
+        outline: 1px solid var(--color-primary);
```

Comment:
It's a 2px border in UX design, not 1px border + 1px outline
<img width="852" alt="image" src="https://user-images.githubusercontent.com/12833674/163340429-59743166-2dbb-42aa-8383-7ccbe5cafab1.png">

Reply:
updated, and added hover shadow effect.